src/data/faq.json

```
@@ -101,6 +101,154 @@
+                        "active": true,
+                        "textblock": [
+                            "All persons who are entitled to a vaccination may also have a digital proof of vaccination available in addition to the paper-based vaccination certificate." ,
+                            "If necessary, you can obtain a corona vaccination certificate from the doctor's office or vaccination center where the vaccination was carried out. Simply ask on site. However, vaccination centers and medical practices are not obliged to issue vaccination certificates." ,
+                            "Various ways are being explored to create digital vaccination certificates for Corona vaccinations afterwards. In principle, the subsequent issuance should take place where one has been vaccinated. If the appropriate contact details are available in the vaccination centers, the QR codes should be sent automatically by mail as far as possible. In addition, doctors and pharmacists could subsequently provide proof of vaccination."
+                        ]
+                    },
+                    {
+                        "title": "Can I use a complete vaccination status in the app as a travel permission?",
+                        "anchor": "vac_cert_privileges_travel",
+                        "active": true,
+                        "textblock": [
+                            "For example, whether you are allowed to travel with completed vaccination protection, depends on the regulations of the respective country.",
+                            "Among other things, you can use this   <a href='https://reopen.europa.eu/en' target='_blank' rel='noopener noreferrer'>tool</a> in order to plan your journey within the EU."
+                        ]
+                    },
+                    {
+                        "title": "How does digital proof of vaccination work?",
+                        "anchor": "vac_cert_how",
+                        "active": true,
+                        "textblock": [
+                            "The digital proof of vaccination is generated in the doctor's pratice or in a vaccination center. After entering or transferring the data, you will be created a vaccination certificate token (QR code), which you can scan directly with the Corona-Warn-App (CWA) or can be handed out on a paper printout and later scanned with the CWA.  You can then manage your digital vaccination certificates on your smartphone."
+                        ]
+                    },
+                    {
+                        "title": "Can I show the QR code without an internet connection?",
+                        "anchor": "vac_cert_internet",
+                        "active": true,
+                        "textblock": [
+                            "Yes, indeed. As soon as the QR code is displayed in the Corona-Warn-App, you can show it at any time without being connected to the internet."

```

Remove "indeed" to align with German. 
```suggestion
                            "Yes. As soon as the QR code is displayed in the Corona-Warn-App, you can show it at any time without being connected to the internet."
```